tests/ssg_test_suite/common.py

```diff
@@ -18,6 +18,38 @@
     "-o", "UserKnownHostsFile=/dev/null",
 )
 
+# Keys are strings used in test scenario header in 'platform' list
+# CentOS CPEs are added to allow testing RHEL profiles on CentOS VMs
+TEST_PLATFORMS = {
+    "Red Hat Enterprise Linux 6": [
+        "cpe:/o:redhat:enterprise_linux:6",
+        "cpe:/o:centos:centos:6"
+    ],
+    "Red Hat Enterprise Linux 7": [
+        "cpe:/o:redhat:enterprise_linux:7",
+        "cpe:/o:centos:centos:7"
+    ],
+    "Red Hat Enterprise Linux 8": [
+        "cpe:/o:redhat:enterprise_linux:8",
+        "cpe:/o:centos:centos:8"
+    ],
+    "Fedora": [
+        "cpe:/o:fedoraproject:fedora:28",
+        "cpe:/o:fedoraproject:fedora:29",
+        "cpe:/o:fedoraproject:fedora:30"
```

Comment:
Would it be possible to source these cpes from a place where we already track it?
For example the `product.yml`?

Reply:
This mapping of product name to cpe could be outsourced to `ssg/products.py`